### ISSUE TITLE
monaco

### ISSUE THREAD

**Issue by buddy-codes** [lafkpages/XL-Replit#30]
monaco

**Comment by DillonB07** [comment]
Absolutely not.

**Comment by DillonB07** [comment]
That's a terrible idea.

**Comment by buddy-codes** [comment]
fuck off

**Comment by DillonB07** [comment]
Monaco isn't great.

**Comment by buddy-codes** [comment]
fuck off

**Comment by DillonB07** [comment]
Codemirror > Monaco

**Comment by buddy-codes** [comment]
fuck fuck fuck fuck fuck

**Comment by DillonB07** [comment]
@lafkpages make this optional.

**Comment by lafkpages** [comment]
Of course, it would be optional, it would be a setting in the popup. But I'm definitely adding it.

**Comment by DillonB07** [comment]
Ok, what is an actual reason for adding it

**Comment by DillonB07** [comment]
I guess we could have proper Emmet, but you'd need to add that as well.

**Comment by lafkpages** [comment]
Monaco is just good

**Comment by DillonB07** [comment]
Codemirror makes more sense for Replit, but honestly one isn't really better than the other. Replit had extremely valid reasons for switching to Codemirror.
Also bear in mind that this could break extensions when the codemirror api releases.

**Comment by lafkpages** [comment]
Yes it could, but I could add my own API for Monaco then.

**Comment by buddy-codes** [comment]
dillon fuck off

**Comment by buddy-codes** [comment]
like seriously lmao 🤓

**Comment by Awokens** [comment]
use stackoverflow 🤓

**Comment by Awokens** [comment]
i am a full stack software engineer 🤓 i make videos about drinking coffee in day in life software engineer 🤓

**Comment by buddy-codes** [comment]
![image](https://user-images.githubusercontent.com/55510116/234115702-20767236-446c-4e8a-a80a-93c7032ccd10.png)

**Comment by buddy-codes** [comment]
pov you

**Comment by buddy-codes** [comment]
im kidding

**Comment by Awokens** [comment]
what's ur framework?

**Comment by buddy-codes** [comment]
sveltekit

**Comment by connordennison** [comment]
@buddy-codes Please make sure you're following the [GitHub Community Guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines) - it's important to be constructive here!

**Comment by buddy-codes** [comment]
kys

**Comment by buddy-codes** [comment]
kiding

**Comment by lafkpages** [comment]
What happened to this issue 💀

**Comment by buddy-codes** [comment]
shut the fuck up

**Comment by datkat21** [comment]
@lafkpages fun  vc discussion in beans toast room

**Comment by buddy-codes** [comment]
![image](https://user-images.githubusercontent.com/55510116/234116028-d8be939d-b38a-48e5-9d96-684d9a9ab934.png)

**Comment by buddy-codes** [comment]
![image](https://user-images.githubusercontent.com/55510116/234116042-f3c5fee5-7993-4a70-b3fa-41e85af1e33e.png)

**Comment by buddy-codes** [comment]
![image](https://user-images.githubusercontent.com/55510116/234116044-d4fd0ce5-98ec-4a06-8f28-1ac7342ab8dd.png)
![image](https://user-images.githubusercontent.com/55510116/234116045-8eaa1be9-2b3b-4caa-a877-d8752587108e.png)
![image](https://user-images.githubusercontent.com/55510116/234116047-0f2d117b-3cbe-4009-a7d0-f6d1d2b9be40.png)

**Comment by buddy-codes** [comment]
![image](https://user-images.githubusercontent.com/55510116/234116067-9f44d6b5-4d84-426f-9e6a-b33546ef9c47.png)
![image](https://user-images.githubusercontent.com/55510116/234116068-581cc9df-c88c-4316-b9d8-0bcef1bae11f.png)
![image](https://user-images.githubusercontent.com/55510116/234116069-b4e76963-4c8b-4f40-bec4-c6f7f15f62d7.png)
![image](https://user-images.githubusercontent.com/55510116/234116071-eb9e35ff-00dd-4824-b152-8ee70fa84d95.png)
![image](https://user-images.githubusercontent.com/55510116/234116073-6cb7edbd-dc6d-4d72-a42d-a038606a0146.png)
![image](https://user-images.githubusercontent.com/55510116/234116075-95c806c7-09ec-4860-bcb8-743004d75c1d.png)
![image](https://user-images.githubusercontent.com/55510116/234116078-ba063d79-e449-4604-81aa-3f5f19b11039.png)
![image](https://user-images.githubusercontent.com/55510116/234116080-85e06877-ce55-4f96-ac0b-088104ea417e.png)
![image](https://user-images.githubusercontent.com/55510116/234116076-28bc242a-6fdd-468c-adfa-88bfd4230feb.png)
![image](https://user-images.githubusercontent.com/55510116/234116077-ebf64a53-5f2d-4bb2-a20b-cc299604af86.png)
![image](https://user-images.githubusercontent.com/55510116/234116082-1bc5e6ec-c99a-4c32-bb06-2d6df02375e3.png)
![image](https://user-images.githubusercontent.com/55510116/234116083-23b815d9-00f7-4f46-a443-48880094a0d5.png)
![image](https://user-images.githubusercontent.com/55510116/234116079-05a49cdd-54ef-46cb-994b-7da2c536dbbb.png)
![image](https://user-images.githubusercontent.com/55510116/234116087-1dff97ab-b9e6-44f4-b410-79e6d1d9ed5c.png)
![image](https://user-images.githubusercontent.com/55510116/234116086-c05163d5-7fe7-4064-80c6-f4c65a55eb9c.png)
![image](https://user-images.githubusercontent.com/55510116/234116081-ad54d6d7-c69f-4b49-8cf6-5e1a3f3df708.png)
![image](https://user-images.githubusercontent.com/55510116/234116088-42082be4-edbb-459b-9cd8-38bf74f48b38.png)
![Uploading image.png…]()
![Uploading image.png…]()

**Comment by lafkpages** [comment]
Longest issue on GitHub fr

**Comment by datkat21** [comment]
absolute issue

**Comment by lafkpages** [comment]
![Uploading D4DC350A-A591-49C0-B303-E0EE80990D97.jpeg…]()

**Comment by connordennison** [comment]
@buddy-codes There will be [severe consequences](https://youtu.be/awyN45qZmvs) granted you fail to follow the guidelines.

**Comment by ErikoXDev** [comment]
🤓🤓🤓🤓🤓🤓
